.rubocop.yml

```diff
@@ -9,6 +9,7 @@ AllCops:
   NewCops: enable
   SuggestExtensions: false
   Exclude:
+    - 'db/migrate/*'
```

Comment:
I added this just for Metrics/MethodLength: but it is probably better to exclude the db/migrate folder globally as you do here 👍  Maybe we can just delete it for Metrics/MethodLength: to avoid duplication.